content/docs/install/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Installation
+description: How to install and configure your Keptn environment
+weight: 30
+icon: concepts
+---
+
+The steps to install and configure your Keptn cluster are:
+
+1. [Create or bring a Kubernetes cluster](k8s)
+    * Check [Kubernetes support and cluster size](k8s-support)
+    to ensure that your version of Kubernetes is compatible
+    with the Keptn version you are installing
+    and that you have adequate resources.
+
+2. Decide [how you want to access Keptn](access).
+   Kubernetes supports expsing Keptn via a LoadBalancer, a NodePort, an Ingress
+   or using a Port-forward.
+
+3. Install [Keptn CLI](cli-install)
+
+4. Install Keptn
+    * Using a [Helm chart](helm-install), all on one cluster
+    * [Manually](../0.16.x/operate/install/#install-keptn),
```

Comment:
No, that very deliberately points to 0.16.x because I think that is the last release that supports the keptn install commands, right?  We need to talk about how we want to handle that.